### ISSUE TITLE
Dart Syntax Highlighting issue with `as` keyword.

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#677]
(Original issue 970 created by caffinatedmonkey on 2014-03-25T22:06:42.373920+00:00)

There seems to be an issue with the Dart lexer. For example, in the following code, `as` should not be highlighted anywhere. Strings don't seem to have an issue but function names, library names and variable names do.
```
#!dart

library ast;

void asthat();

var tt = 'as';
var assa = 'error';
```
Here is the pygments demo: http://pygments.org/demo/304303/?style=monokai

**Comment by Anteru** [comment]
(Original issue 970 last updated on 2014-04-14T16:38:17.805904+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-04-14T16:38:17.795706+00:00)

Closes #970: fix overly zealous recognition of "as" as an operator.

→ <<cset 4bd01fd50e81>>
